src/components/Typography/Editorial.js

```diff
@@ -158,6 +159,7 @@ export const Emphasis = ({ children, attributes, ...props }) => (
 
 const link = css({
   textDecoration: 'underline',
+  textDecorationSkip: 'ink',
```

Comment:
This repeated combo of textDecoration styles could probably go to lib/styleMixins.js